jupyter/Dockerfile

```diff
@@ -5,19 +5,21 @@ FROM arynai/sycamore-importer:$TAG
 
 # TODO: eric - remove this once we switch other containers over to non-root
 # Note user/group should be 1000 to match with likely defaults on hosts.
+USER root
 COPY jupyter/docker-root-setup.sh /root
 RUN /bin/bash /root/docker-root-setup.sh
 COPY jupyter/sudoers /etc/sudoers
 COPY jupyter/fixuser.py /root/fixuser.py
 
-USER 1000
+# User 'app' must be id number 1000 (the default) for all to work properly.
+USER app
 RUN poetry run pip install notebook ipywidgets 'opensearch-py>=2.4' && rm -rf /app/.cache
 RUN mkdir -p /app/work/docker_volume /app/work/bind_dir /app/work/examples
 RUN touch /app/work/AAA_SEE_README_FOR_PERSISTENT_DATA_DIRECTORIES
 COPY jupyter/run-jupyter.sh ./
 COPY jupyter/README.md ./work
-COPY sycamore_local_dev_example.ipynb ./work/examples
-RUN perl -i -pe 's/localhost/opensearch/ if /9200/;s,tmp/sycamore/data,/app/work/docker_volume,' /app/work/examples/sycamore_local_dev_example.ipynb
+COPY notebooks/jupyter_dev_example.ipynb ./work/examples
```

Comment:
This would have been a good thing to have in another PR; looks like we broke Docker builds at head accidentally during the merge.